compiler-rt/lib/scudo/standalone/vector.h

```diff
@@ -9,6 +9,7 @@
 #ifndef SCUDO_VECTOR_H_
 #define SCUDO_VECTOR_H_
 
+#include "common.h"
```

Comment:
I think the file is missing `#include "internal_defs.h"`

Reply:
Is this because of the `size_t`? I don't see this defined in `internal_defs.h`, but if it's for something else, I will add it.

Reply:
Even though we don't see any compilation error but I think it's missing some header there. (mem_map.h is not enough obviously). Some required headers may be included indirectly by mem_map.h